lib/cocoapods/generate/installer.rb

```diff
@@ -148,7 +153,10 @@ def perform_post_install_steps(app_project, installer)
           remove_script_phase_from_target(native_app_target, 'Check Pods Manifest.lock')
 
           pod_target = installer.pod_targets.find { |pt| pt.platform.name == native_app_target.platform_name && pt.pod_name == spec.name }
-          raise "unable to find a pod target for #{native_app_target} / #{spec}" unless pod_target
+          raise "Unable to find a pod target for #{native_app_target} / #{spec}" unless pod_target
+
+          native_app_target.source_build_phase.clear
```

Comment:
if we are doing incremental installation and this code runs we need to ensure we dont double add sources or resources to the build phase of the app "user" target.